security/calico-enterprise/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Calico Enterprise
-description: Learn about value-added features in our commercial product, Calico Enterprise. 
+description: Learn about the production-grade features for implementing a CaaS platform in our commercial product, Calico Enterprise. 
```

Comment:
"value-added" and "production-grade" are different concepts I think.  Calico open source is already production grade.  I think the idea of "value-added" was to say there's additional stuff you might want.

Reply:
Reverted to "value-added"